docs/workflow/testing/viewing-ci-runtime-test-failures.md

```diff
@@ -0,0 +1,27 @@
+# Finding the failed Runtime Tests in CI
+
+The process for finding out which tests actually failed when a runtime test leg
+fails is not straight forward.
+
+Suppose for example you see on github that this leg failed:
+
+```
+runtime (Mono Pri0 Runtime Tests Run Linux arm release)
+```
+
+To find the actual test failure click `details`
+
+Then `View more details on Azure Pipelines`.
+
+Near the top, you will see something like
+
+`dnceng / public / Pipelines / runtime / <build id>`
```

Comment:
Just making sure this is publicly viewable by the community or is internal info to employees ?

Reply:
it's public